### PR TITLE
GP-05v2 Successed with JWT tokens

### PR DESCRIPTION
**Pull request by ShefardPT**

### CODE REVIEW

File: src/GeoPing.Api/SeedData.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+namespace GeoPing.Api
+{
+    public static class SeedData
+    {
+        public static void Initialize(IServiceProvider serviceProvider)
+        {
+            var context = serviceProvider.GetRequiredService<ApplicationDbContext>();
+            context.Database.EnsureCreated();
+            if (!context.Users.Any())
```
Comment (jokerosky) [inline src/GeoPing.Api/SeedData.cs]: Users should be created via UserManager like in natural order, without direc inserting records into database!

File: src/GeoPing.Api/Startup.cs [hunk 2]
```diff
@@ -14,37 +14,48 @@
...
+            _environment = environment;
+
+
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(environment.ContentRootPath)
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                .AddJsonFile($"appsettings.{environment.EnvironmentName}.json", optional: true);
+
+            builder.AddEnvironmentVariables();
+            _configuration = builder.Build();
```
Comment (jokerosky) [inline src/GeoPing.Api/Startup.cs]: Doesn't it build configuration also using enviroment files by default, should we do i manually?

File: src/GeoPing.Api/Constants.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GeoPing.Api
+{
+    public static class Constants
```
Comment (jokerosky) [inline src/GeoPing.Api/Constants.cs]: still hardcoded in static manner, can we use Configuration for this?

File: src/GeoPing.Api/SeedData.cs (hunk 1 above)
Comment (ShefardPT) [reply src/GeoPing.Api/SeedData.cs]: Done

File: src/GeoPing.Api/Startup.cs (hunk 2 above)
Comment (ShefardPT) [reply src/GeoPing.Api/Startup.cs]: As I know, it builds config using enviroment name and takes unfounded values from appsettings, am I wrong?

File: src/GeoPing.Api/Constants.cs (hunk 3 above)
Comment (ShefardPT) [reply src/GeoPing.Api/Constants.cs]: In progress

**Pull request merged**